cmd/awscollector-prometheus/Dockerfile

```diff
@@ -0,0 +1,54 @@
+################################
+#	Building Stage         #	
+#			       #	
+################################
+FROM alpine:latest as build 
+
+# update cert
```

Comment:
I thought we will only update the prom config in the existing Dockerfile `cmd/awscollector/Dockerfile` instead of creating a new one?

Reply:
@mxiamxia Sorry, I pushed the wrong ref... Just fixed that.